MSVC Unvanquished CI is broken, maybe Appveyor updated some python

See: https://ci.appveyor.com/project/UnvanquishedDevelopment/unvanquished/builds/50559706

```
-- Found Python: C:/Python313/python.exe (found version "3.13.0") found components: Interpreter
-- Using CBSE Python executable: C:/Python313/python.exe
Traceback (most recent call last):
  File "<string>", line 1, in <module>
    import jinja2, yaml, collections, argparse, sys, os.path, re
ModuleNotFoundError: No module named 'jinja2'
CMake Error at tools/cbse/CBSE.cmake:48 (message):
  Missing dependences for CBSE generation.  Please ensure you have python ≥
  2, python-yaml, and python-jinja installed.
                               Use pip install -r tools/cbse/requirements.txt to install
Call Stack (most recent call first):
  CMakeLists.txt:173 (CBSE)
-- Configuring incomplete, errors occurred!
```

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -40,7 +40,7 @@ before_build:
     --sub-ref=%APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH%:has=/sync$
     --print
   - python --version
-  - pip install -r tools/cbse/requirements.txt --upgrade
+  - python -m pip install -r tools/cbse/requirements.txt --upgrade
 
 build_script:
   - cmd: cmake --version

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -45,7 +45,11 @@ before_build:
 build_script:
   - cmd: cmake --version
   - cmd: call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars32.bat"
+  # We don't need iteration; the for loop executes only once. But the function for resolving
+  # something in PATH (like 'which') is in the FOR command
+  - cmd: FOR %%p IN (python.exe) DO set firstPython=%%~$PATH:p
   - cmd: cmake -DUSE_PRECOMPILED_HEADER=0 -DUSE_WERROR=1 -DBE_VERBOSE=1 -G "NMake Makefiles"
-         -DBUILD_CLIENT=0 -DBUILD_TTY_CLIENT=0 -DBUILD_SERVER=0 -DBUILD_GAME_NACL=0 -DCMAKE_BUILD_TYPE=Debug
+         -DBUILD_CLIENT=0 -DBUILD_TTY_CLIENT=0 -DBUILD_SERVER=0 -DBUILD_GAME_NACL=0
+         -DCMAKE_BUILD_TYPE=Debug -DCBSE_PYTHON_PATH=%firstPython%
          -H. -Bbuild
   - cmd: cmake --build build

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -28,22 +28,22 @@ strategy:
     Linux GCC Native DLL:
       VM_IMAGE: 'ubuntu-20.04'
       INSTALL_DEPS: 'sudo apt-get update && sudo apt-get -y -q --no-install-recommends install libfreetype6-dev liblua5.2-dev python-yaml python-jinja2 ninja-build gcc-8 g++-8'
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8 -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8 -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
       BUILDER_OPTIONS: '-j$(nproc) -k 10'
     Linux Clang Native DLL:
       VM_IMAGE: 'ubuntu-20.04'
       INSTALL_DEPS: 'sudo apt-get update && sudo apt-get -y -q --no-install-recommends install libfreetype6-dev liblua5.2-dev python-yaml python-jinja2 ninja-build'
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
       BUILDER_OPTIONS: '-j$(nproc) -k 10'
     Linux NaCl:
       VM_IMAGE: 'ubuntu-20.04'
       INSTALL_DEPS: 'sudo apt-get update && sudo apt-get -y -q --no-install-recommends install python-yaml python-jinja2 ninja-build'
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DBUILD_GAME_NATIVE_DLL=0 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=1'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DBUILD_GAME_NATIVE_DLL=0 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=1'
       BUILDER_OPTIONS: '-j$(nproc) -k 10'
     Mac:
       VM_IMAGE: 'macOS-12'
       INSTALL_DEPS: 'python3 -m pip install -r tools/cbse/requirements.txt'
-      CMAKE_OPTIONS: '-DDAEMON_CBSE_PYTHON_PATH=$(which python3) -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-DCBSE_PYTHON_PATH=$(which python3) -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
       BUILDER_OPTIONS: '-j$(sysctl -n hw.logicalcpu)'
 
 pool:

--- a/tools/cbse/CBSE.cmake
+++ b/tools/cbse/CBSE.cmake
@@ -25,12 +25,12 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Allow overriding the python path with a cached variable.
-set(DAEMON_CBSE_PYTHON_PATH "NOTFOUND" CACHE FILEPATH "Python executable for CBSE code generation")
-if (NOT DAEMON_CBSE_PYTHON_PATH OR NOT EXISTS ${DAEMON_CBSE_PYTHON_PATH})
+set(CBSE_PYTHON_PATH "NOTFOUND" CACHE FILEPATH "Python executable for CBSE code generation")
+if (NOT CBSE_PYTHON_PATH OR NOT EXISTS ${CBSE_PYTHON_PATH})
     find_package(Python REQUIRED)
-    set(DAEMON_CBSE_PYTHON_PATH "${Python_EXECUTABLE}" CACHE FILEPATH "Python executable for CBSE code generation" FORCE)
+    set(CBSE_PYTHON_PATH "${Python_EXECUTABLE}" CACHE FILEPATH "Python executable for CBSE code generation" FORCE)
 endif()
-message(STATUS "Using CBSE Python executable: ${DAEMON_CBSE_PYTHON_PATH}")
+message(STATUS "Using CBSE Python executable: ${CBSE_PYTHON_PATH}")
 
 function(maybe_add_dep target dep)
     if (TARGET ${target})
@@ -42,7 +42,7 @@ function(CBSE target definition output)
     # Check if python has all the dependencies
     # TODO: Execute pip directly here and install them
     execute_process(
-        COMMAND ${DAEMON_CBSE_PYTHON_PATH} -c "import jinja2, yaml, collections, argparse, sys, os.path, re"
+        COMMAND ${CBSE_PYTHON_PATH} -c "import jinja2, yaml, collections, argparse, sys, os.path, re"
         RESULT_VARIABLE RET)
     if (NOT RET EQUAL 0)
         message(FATAL_ERROR "Missing dependences for CBSE generation. Please ensure you have python ≥ 2, python-yaml, and python-jinja installed.
@@ -66,7 +66,7 @@ function(CBSE target definition output)
                 ${CMAKE_SOURCE_DIR}/tools/cbse/templates/Helper.h
         WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/tools/cbse
         COMMAND
-                ${DAEMON_CBSE_PYTHON_PATH}
+                ${CBSE_PYTHON_PATH}
                 ${CMAKE_SOURCE_DIR}/tools/cbse/CBSE.py
                 -s -o
                 "${output}"

--- a/tools/cbse/CBSE.cmake
+++ b/tools/cbse/CBSE.cmake
@@ -40,13 +40,12 @@ endfunction()
 
 function(CBSE target definition output)
     # Check if python has all the dependencies
-    # TODO: Execute pip directly here and install them
     execute_process(
         COMMAND ${CBSE_PYTHON_PATH} -c "import jinja2, yaml, collections, argparse, sys, os.path, re"
         RESULT_VARIABLE RET)
     if (NOT RET EQUAL 0)
         message(FATAL_ERROR "Missing dependences for CBSE generation. Please ensure you have python ≥ 2, python-yaml, and python-jinja installed.
-                             Use pip install -r tools/cbse/requirements.txt to install")
+                             Use python -m pip install -r tools/cbse/requirements.txt to install")
     endif()
     set(GENERATED_CBSE ${output}/backend/CBSEBackend.cpp
                        ${output}/backend/CBSEBackend.h

